src/main/java/com/intel/genomicsdb/GenomicsDBImporter.java

```diff
@@ -360,15 +145,6 @@ public GenomicsDBImporter(Map<String, FeatureReader<VariantContext>> sampleToRea
      * @param sampleToReaderMap               Feature Readers objects corresponding to input GVCF files
      * @param mergedHeader                    Set of VCFHeaderLine from the merged header across all input files
      * @param chromosomeInterval              Chromosome interval to traverse input VCFs
-     * @param workspace                       TileDB workspace
-     * @param arrayname                       TileDB array name
-     * @param vcfBufferSizePerColumnPartition vcfBufferSizePerColumnPartition in bytes
-     * @param segmentSize                     segmentSize in bytes
-     * @param lbRowIdx                        Smallest row idx which should be imported by this object
-     * @param ubRowIdx                        Largest row idx which should be imported by this object
-     * @param useSamplesInOrderProvided       if true, don't sort samples, instead use in the the order
-     *                                        provided
-     * @param failIfUpdating                  if true, fail if updating an existing array
      * @param rank                            Rank of object - corresponds to the partition index in the loader
```

Comment:
Javadoc comment for importConfiguration? Or maybe I'm missing the line because of the way Github is displaying the diff